docs/usage.rst

```diff
@@ -387,8 +387,14 @@ accept unsigned content with ``accept_untrusted_content=True``:
     ...                     sender.request_header,
```

Comment:
While you're in here could you fix the "unsigned" part of the docs? I think it should read more like:

The :class:`mohawk.Receiver` must also be constructed to accept content without a declared hash using accept_untrusted_content=True: